internal/server/v0/placestatvar/golden/get_place_stat_vars/california.json

```diff
@@ -35135,7 +35131,6 @@
         "dc/0hry4lccfhcgf",
         "dc/0hsbkjer9x9jf",
         "dc/0hslbewgd1h6c",
-        "dc/0hsryns4ejntb",
```

Comment:
These are variables generated by the Sahie import poverty line changes which was reverted later to use the existing variables as part of CL#635471476. 

Series with the human readable SV is present in prod : https://screenshot.googleplex.com/5A34zFCkJVwXBNL.
Deleted :  https://screenshot.googleplex.com/7xtQsGFmrYrmnqe

After reverting, the import was added to frequent and rebuilt but the  us census cache was not rebuilt. Hence the new dc/ variables generated were part of the last release which is getting deleted now.